src/solve.jl

```diff
@@ -63,8 +63,6 @@ function solve!(system::InitialValueProblem{<:Union{AbstractContinuousSystem,
                )::AbstractSolution
     options = init(op, system, options_input)
 
-    # coordinate transformation
-    options[:transformation_matrix] = nothing
```

Comment:
Please keep this line here. It belongs to the code below as a default value and is independent of the continuous algorithm. You can add it as an `else` case if that makes it clearer. Or we refactor the whole code to a function.

Reply:
i get the idea: this option is expected to be shared by all continuous algorithms with linear affine dynamics (we have essentially 1 now) so it can be pulled out form the init code. Same for the block

```julia
    if options[:coordinate_transformation] != ""
        info("Transformation...")
        tic()
        (system, transformation_matrix) =
            transform(system, options[:coordinate_transformation])
        tocc()
        options[:transformation_matrix] = transformation_matrix
        invariant = options[:coordinate_transformation] * invariant
    end
```

agreed to revert the change. eventually we can move the code to an `init_generic` function that is run before or after `init` for the given algorithm.

Reply:
Yes, ideally we would use different interfaces for "purely continuous" (1 shot) and "call continuous from the hybrid loop" because you would want to do this only once.